lib/client.js

```diff
@@ -311,7 +341,18 @@ Client.prototype = {
     }.bind(this));
   },
 
+  instance: function(instanceGuid) {
```

Comment:
Could have avoided each sub client having the `instanceClients` and this guard by making `instances` a static method, like `init`?